tests/integration/openapi3-express.spec.ts

```diff
@@ -148,14 +146,14 @@ describe('OpenAPI3 Express Server', () => {
     bodyModel.stringMax10Lenght = 'abcdefghijk';
     bodyModel.stringMin5Lenght = 'abcd';
     bodyModel.stringPatternAZaz = 'ab01234';
+    bodyModel.quotedStringPatternA = 'A';
 
     bodyModel.arrayMax5Item = [0, 1, 2, 3, 4, 6, 7, 8, 9];
     bodyModel.arrayMin2Item = [0];
     bodyModel.arrayUniqueItem = [0, 0, 1, 1];
-    bodyModel.intersection = { value1: '' } as any;
```

Comment:
Why remove these?

Reply:
Was not intended. I have added a commit fix unintended changes in this file.